### PR TITLE
feat: charts legend resize drag-handle improvements #2055

### PR DESCRIPTION
**Pull request by Priyanshu44**
When the user toggles the legend feature in the style panel in configuration, the vertical slider is also hidden along with legend container.

Verification:
[slider-hide-legend-toggle.webm](https://github.com/awslabs/iot-app-kit/assets/81667589/131c9f0c-ee1b-4cbf-b871-5089980bf8c9)

### CODE REVIEW

File: packages/react-components/src/components/time-sync/viewportAdapter.spec.ts [hunk 1]
```diff
@@ -151,8 +151,8 @@ describe('getViewportStartOnBackwardRelative', () => {
       unit: 'minute',
       type: 'relative',
     });
-
-    expect(currentDate.getTime() - newDate.getTime()).toEqual(300000);
+    // Failing in PR
```
Comment (mnischay) [inline packages/react-components/src/components/time-sync/viewportAdapter.spec.ts:154]: on re-run these are passing, just re-run the tests

Comment (corteggiano) [reply packages/react-components/src/components/time-sync/viewportAdapter.spec.ts:154]: +1 they are faulty tests not related to your change, will be fixed soon

File: packages/react-components/src/components/chart/baseChart.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (mnischay) [inline packages/react-components/src/components/chart/baseChart.spec.tsx:1]: There is already a tests folder, mind moving this to file

**Review by mnischay — COMMENTED** [review]
Code changes LGTM, just move the test file and renmae the folder svg to assets , should take less than 5 mins

**Comment by Priyanshu44** [comment]
Closing this PR as my branch got messed up while rebase rc, raising another PR.

**Pull request closed without merging**